Server/src/main/kotlin/net/horizonsend/ion/server/listeners/bukkit/PlayerJoinListener.kt

```diff
@@ -1,14 +1,37 @@
 package net.horizonsend.ion.server.listeners.bukkit
 
+import net.kyori.adventure.text.format.TextColor
+import net.kyori.adventure.text.minimessage.MiniMessage
+import org.bukkit.GameMode
+import org.bukkit.Material
 import org.bukkit.event.EventHandler
 import org.bukkit.event.EventPriority
 import org.bukkit.event.Listener
 import org.bukkit.event.player.PlayerJoinEvent
+import org.bukkit.inventory.ItemStack
 
 class PlayerJoinListener : Listener {
 	@EventHandler(priority = EventPriority.NORMAL)
 	@Suppress("Unused")
 	fun onPlayerJoinEvent(event: PlayerJoinEvent) {
 		event.joinMessage(null)
+
+		if(!event.player.hasPlayedBefore() && event.player.gameMode == GameMode.SURVIVAL){
+			val armor: Array<ItemStack?> = arrayOf()
+			armor[0] = ItemStack(Material.CHAINMAIL_BOOTS, 1)
+			armor[1] = ItemStack(Material.CHAINMAIL_LEGGINGS, 1)
+			armor[2] = ItemStack(Material.CHAINMAIL_CHESTPLATE, 1)
+			armor[3] = ItemStack(Material.CHAINMAIL_HELMET, 1)
+
+			event.player.inventory.armorContents = armor
+
+			val kitstarterasking = ItemStack(Material.PAPER)
```

Comment:
Why not just a chat message?

Reply:
I want players to remember it, and a chat message will get burried, but a piece of paper is harder to miss/lose

Reply:
Mind seting a PDC value so we can keep track of these items and nuke them later?

Reply:
Sure, gotto figure out wtf a pdc value is tho

Reply:
Oh persistent data container

Reply:
See `net.horizonsend.ion.server.listeners.bukkit.ChunkLoadListener` lines 24, 34, and 105-109.

Reply:
Not to worry, I found a stack overflow answer

Reply:
:kekw:

Reply:
Will I do NamespacedKey(event.player.server.name, "SpawnPaper")
Because I cant do NamespacedKey(IonServer, "SpawnPaper")

Reply:
`IonServer.plugin`?

Reply:
Doesnt let me

Reply:
What?

Reply:
I cant do IonServer.plugin

Reply:
Look in dev-discussion

Reply:
Oh... that doesn't exist yet. Put plugin as a constructor parameter, see ChunkLoadListener.

Reply:
I did